spec/rails_helper.rb

```diff
@@ -8,6 +8,8 @@
 require "rspec/rails"
 require "pundit/rspec"
 
+Capybara.server = :puma, { Silent: true }
```

Comment:
It's just a small cosmetic to avoid having this output 

```
$ rspec spec/system

Randomized with seed 53334
.Capybara starting Puma...
* Version 3.12.0 , codename: Llamas in Pajamas
* Min threads: 0, max threads: 4
* Listening on tcp://127.0.0.1:44779
..
```

If you think it's unnecessary I can drop that.